integration_test/soak_test/cmd/launcher/main.go

```diff
@@ -106,6 +106,12 @@ func mainErr() error {
 		Labels: map[string]string{
 			"ttl": strconv.Itoa(int(parsedTTL / time.Minute)),
 		},
+		Metadata: map[string]string{
+			// This is to avoid b/295165549 on Windows and also
```

Comment:
It might be better to describe the problem here, rather than reference an internal link. For example:
```
// This is to avoid Windows updates and reboots, and also
// to avoid throughput blips when the OS Config agent
// runs periodically.
```
Also, why not `"osconfig-disabled-features": "tasks"` [1] instead?

[1] https://cloud.google.com/compute/docs/manage-os#disable-features